exercises/Set3a.hs

```diff
@@ -236,7 +236,7 @@ multiCompose fs = todo
 --   multiApp sum [(1+), (^3), (+2)] 1  ==>  6
 --   multiApp reverse [tail, take 2, reverse] "foo" ==> ["oof","fo","oo"]
 --   multiApp concat [take 3, reverse] "race" ==> "racecar"
---   multiApp id [head, (!!2), last] "axbxc" ==> "abc"
+--   multiApp id [head, (!!2), last] "axbxc" ==> ["a","b","c"]
```

Comment:
Since `head`, `(!!2)` and `last` return characters when applied to a string, the result is `['a','b','c']`, which GHCi renders as `"abc"`. Do you think it'd clarify this if I changed the return value of this example to something like:

> `['a','b','c']`, in other words, `"abc"`

Reply:
Thanks for the clarification! I think it would be helpful to add your suggested comment for people coming from other languages